### ISSUE TITLE
Can we push our proxy docker images to a public URL on each master push?

### ISSUE THREAD

**Issue by arschles** [gomods/athens#735]
**Is your feature request related to a problem? Please describe.**

Right now, we have an Athens running on a public URL (https://microsoftgoproxy.azurewebsites.net). It was manually deployed before GopherCon 2018 (thanks @joshgav and @marstr!) and hasn't been pushed since. It would be nice to have an up-to-date image from master on some public URL so that people can test out the latest Athens

**Describe the solution you'd like**

I'd like our CI/CD system to push master to our public URL on every merge to master

**Describe alternatives you've considered**

Nothing

**Additional context**

I mentioned this in the [October 4, 2018 Dev Meeting](https://docs.gomods.io/install/). It would be nice to have for v0.1.0, but not required

**Comment by chriscoffee** [comment]
As @robjloranger said, first enable continuous deployment by either running the command in the docs or going to **container settings** (the docs are out of date it seems) and enable continuous deployment. Then just grab the webhook it provides underneath and throw it into https://hub.docker.com/r/gomods/proxy/~/settings/webhooks/ (docs for that [are here](https://docs.docker.com/docker-hub/webhooks/))

This can be done a bunch of other ways, like getting travis to notify the webhook or fucntions but that's definitely the easiest if we've got acess to both Azure and Docker Hub. And probably the least overengineered option for v0.1.0

I made an [example](https://proxy-all-the-things.azurewebsites.net/) off a mirror to test it and it works fine.

**Comment by marpio** [comment]
@arschles do you think we could do this already with the new instance?

**Comment by arschles** [comment]
The deployment is being done from [here](https://github.com/arschles/athens-azure) now, and I'm working on updating the whole architecture based on Terraform and Kubernetes (I wrote about it briefly [here](https://arschles.com/blog/athens-on-azure-kubernetes-service/)). Once that is done, I'll be able to do deployments fairly easily, all via GH actions. At that point, I can pretty easily hook this repo's releases up to that deployment if we want.

I'm going to close this issue now, since it's more related to the other repo
